lib/spack/docs/module_file_support.rst

```diff
@@ -0,0 +1,659 @@
+===============================
+Integration with module systems
+===============================
+
+The use of module systems to manage user environment in a controlled way
+is a common practice at HPC centers that is often embraced also by individual
+programmers on their development machines. To support this common practice
```

Comment:
Suggested reword: 
"that is often embraced also by individual programmers on their development machines" 
-> 
"and is also popular with individual programmers to manage their development machines"

This doesn't change the meaning at all, I just felt that this was slightly clearer. Would you find it useful if I made other suggestions of this sort or are you primarily interested in correctness?